### DISCUSSION TITLE
Artalk请求头像参数错误

### DISCUSSION THREAD

**Discussion by yooongchun** [vuepress-theme-hope/vuepress-theme-hope#3246] [Q&A] [answered]
### Checklist

- [X] I am using **LTS** version of Node.js.
- [X] I am using the **latest v2 version** of VuePress and VuePress official plugins.
- [X] I am using the **latest v2 version** of vuepress-theme-hope and all plugins in this repo.
- [X] I followed the docs and I double checked my configuration.
- [X] I have searched [FAQ](https://theme-hope.vuejs.press/faq/) and I did not find the answer.

### Package name

vuepress-theme-hope

### Which operating system are you using?

- [ ] macOS
- [ ] Windows
- [X] Linux

### Describe the bug

大佬，配置Artalk评论插件之后，Artalk后端配置的参数无法覆盖前端，这是在前端写死的吗？

现在我请求到的前端头像一直用的同一个参数，全部都是灰色头像，查看请求看到前端请求的是：`https://cravatar.cn/avatar/860abc73203dc25ccf303e3eec699154?d=mp&s=240`

这里影响头像的主要是携带的参数`d=mp`

![默认头像](https://cravatar.cn/avatar/860abc73203dc25ccf303e3eec699154?d=mp&s=240)

而我在Artalk后端配置后后端请求的是：
`https://sdn.geekzu.org/avatar/860abc73203dc25ccf303e3eec699154?d=monsterid&s=200`

我后端请求到的头像是：

![个性化头像](https://cravatar.cn/avatar/860abc73203dc25ccf303e3eec699154?d=monsterid&s=240)

我的诉求是：能不能前端配置可被后端覆盖？（我看Artalk是有这个功能的）；或者前端可配置api请求的参数？

**Comment by yooongchun** [comment]
如下图：左侧是前端展示的头像，中间是Artalk的控制中心请求到的头像，右侧是浏览器发送的请求，主要区别在与`d=mp`这个参数，在Artalk后端我配置了`d=monsterid`，但是在前端似乎没有生效。（或许是我弄错了前端的配置项？）

<img width="1529" alt="image" src="https://github.com/vuepress-theme-hope/vuepress-theme-hope/assets/20920603/d2166ecc-3fff-4af3-bd70-39fae809e617">

**Comment by Mister-Hope** [comment, marked as answer]
我觉得这类问题一律应移步artalk，印象中这是个函数选项，应该不支持后端配置应用到前端。我也已经提供了 `defineArtalkConfig` 供你前端配置此选项。

如果你的确认为 `defineArtalkConfig` 没有按照预期工作，新开 issue 并提供有效的 minimal reproduction
